### PR TITLE
Clean contracts docs

### PR DESCRIPTION
**Pull request by riusricardo**
- Remove duplicated instructions for Ink

### CODE REVIEW

**Comment by kaichaosun** [comment]
I may miss some context, but why the deleted docs in this PR are duplicate?
We should also delete the source docs in Crowdin, so translator won't work on the removed docs.
https://github.com/substrate-developer-hub/substrate-developer-hub.github.io#rename-an-existing-document

File: docs/tutorials/adding-a-module-to-your-runtime.md [hunk 1]
```diff
@@ -322,7 +322,7 @@ cargo run --release -- --dev
 
 ## Next Steps
 
-We wont actually go into the details of testing the Contracts module functionality added to this chain, but if you want to try it out, you can follow our instructions for [deploying a contract](contracts/deploying-a-contract.md).
+We wont actually go into the details of testing the Contracts module functionality added to this chain, but if you want to try it out, you can follow our contracts tutorials.
```
Comment (shawntabrizi) [inline docs/tutorials/adding-a-module-to-your-runtime.md]:
```suggestion
We wont actually go into the details of testing the Contracts module functionality added to this chain, but if you want to try it out, you can follow our [contracts tutorials](https://substrate.dev/tutorials).
```

File: docs/ecosystem/contracts/faq.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ---
-title: "Frequently Asked Questions"
+title: "Ink! FAQ"
 ---
 
 This page will answer a number of common questions you may have when starting to build smart contracts for Substrate.
 
-## ink!
+## Ink! FAQ
```
Comment (shawntabrizi) [inline docs/ecosystem/contracts/faq.md]: @robbepop wants to keep the brand `ink!` with a lowercase `i`.

So it is symmetrical with !

File: docs/ecosystem/contracts/faq.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 ---
-title: "Frequently Asked Questions"
+title: "Ink! FAQ"
```
Comment (shawntabrizi) [inline docs/ecosystem/contracts/faq.md]:
```suggestion
title: "ink! FAQ"
```

File: docs/ecosystem/contracts/ink.md [hunk 4]
```diff
@@ -1,11 +1,9 @@
 ---
-title: "Introduction to Smart Contracts on Substrate"
+title: "Ink! Smart Contracts"
```
Comment (shawntabrizi) [inline docs/ecosystem/contracts/ink.md]:
```suggestion
title: "ink! Smart Contracts"
```

File: website/versioned_docs/version-1.0/ecosystem/contracts/faq.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Frequently Asked Questions
+title: Ink! FAQ
```
Comment (shawntabrizi) [inline website/versioned_docs/version-1.0/ecosystem/contracts/faq.md]:
```suggestion
title: ink! FAQ
```

File: website/versioned_docs/version-1.0/ecosystem/contracts/ink.md [hunk 6]
```diff
@@ -1,13 +1,11 @@
 ---
-title: Introduction to Smart Contracts on Substrate
-id: version-1.0-index
-original_id: index
+title: "Ink! Smart Contracts"
```
Comment (shawntabrizi) [inline website/versioned_docs/version-1.0/ecosystem/contracts/ink.md]:
```suggestion
title: "ink! Smart Contracts"
```

File: website/versioned_docs/version-1.0/tutorials/adding-a-module-to-your-runtime.md [hunk 7]
```diff
@@ -281,7 +281,7 @@ cargo build --release
 
 ## Next Steps
 
-We wont actually go into the details of testing the Contract module functionality added to this chain, but if you want to try it out, you can follow our instructions for [deploying a contract](contracts/deploying-a-contract.md).
+We wont actually go into the details of testing the Contracts module functionality added to this chain, but if you want to try it out, you can follow our contracts tutorials.
```
Comment (shawntabrizi) [inline website/versioned_docs/version-1.0/tutorials/adding-a-module-to-your-runtime.md]:
```suggestion
We wont actually go into the details of testing the Contracts module functionality added to this chain, but if you want to try it out, you can follow our [contract tutorials](https://substrate.dev/tutorials).
```

**Comment by shawntabrizi** [comment]
@riusricardo where did you move the other docs which got deleted? For example the ink! types stuff?

**Comment by kaichen** [comment]
This pr broke several links in ink repo and wiki.

The Ink's wiki points document to substrate hub. Documents including coding/deployment/testing are all in this repo. Do you guys plan to move these to another place?

**Comment by riusricardo** [comment]
@kaichen Thanks for letting us know. We'll restructure the Developer Hub. You can find the same instructions on the tutorial. https://substrate.dev/substrate-contracts-workshop/#/0/introduction

**Comment by kaichen** [comment]
@riusricardo Cool, expect the restructure work make the document better 😃

**Comment by shawntabrizi** [comment]
@kaichen please take a look here to follow along:
https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/pull/250

However, in general, we found that the contracts stuff is a bit too much in flux right now to get spots in our documentation. While things are changing more quickly, I think it is best to consolidate and minimize the number of docs we have which teach the same thing. Thus when something breaks, there are less things to update.

In this case, we want to keep the `substrate-contracts-workshop` working end to end, and generally the only path for users who want to explore Substrate Smart Contracts.

**Pull request merged**